<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerPathReconstructionTableWidget</class>
 <widget class="qSlicerWidget" name="qSlicerPathReconstructionTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>195</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="3">
      <widget class="QPushButton" name="RefitPathsButton">
       <property name="toolTip">
        <string>Re-apply the fitting as per the current MarkupsToModel node</string>
       </property>
       <property name="text">
        <string>Refit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="qMRMLNodeComboBox" name="FittingParametersComboBox">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLMarkupsToModelNode</string>
        </stringlist>
       </property>
       <property name="noneEnabled">
        <bool>true</bool>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="FittingParametersLabel">
       <property name="text">
        <string>Fitting parameters</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="qMRMLNodeComboBox" name="PathReconstructionNodeComboBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLMarkupsFiducialNode</string>
        </stringlist>
       </property>
       <property name="showHidden">
        <bool>true</bool>
       </property>
       <property name="noneEnabled">
        <bool>true</bool>
       </property>
       <property name="addEnabled">
        <bool>false</bool>
       </property>
       <property name="removeEnabled">
        <bool>true</bool>
       </property>
       <property name="editEnabled">
        <bool>false</bool>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="ctkColorPickerButton" name="FittingColorPicker">
       <property name="displayColorName">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="PathsTable"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>ctkColorPickerButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../S4/Modules/Loadable/Markups/Resources/qSlicerMarkupsModule.qrc"/>
  <include location="../../../../../S4/Libs/MRML/Widgets/Resources/qMRMLWidgets.qrc"/>
  <include location="../../../../../S4/Base/QTGUI/Resources/qSlicerBaseQTGUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qSlicerPathReconstructionTableWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>PathReconstructionNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerPathReconstructionTableWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>FittingParametersComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
